plc4go/protocols/modbus/readwrite/XmlParserHelper.go

```diff
@@ -51,7 +51,7 @@ func (m ModbusXmlParserHelper) Parse(typeName string, xmlString string, parserAr
 			return nil, err
 		}
 		numberOfValues := uint16(parsedUint1)
-		return model.DataItemParse(utils.NewXmlReadBuffer(strings.NewReader(xmlString)), dataType, numberOfValues)
+		return model.DataItemParse(utils.NewXmlReadBuffer(strings.NewReader(xmlString)), dataType, numberOfValues, 0)
```

Comment:
This is a generated file and will be replaced by an updated version, effectively reverting your changes the next time "mvn install" is executed in the parent.